uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt="" role="presentation"/>
```

Comment:
@CR96 

    the alt text attribute itself was added (it has to be present to pass ADA checks)

Okay, but shouldn't the `alt` attribute be non-blank?